engine/src/main/java/org/pentaho/di/repository/RepositorySecurityProvider.java

```diff
@@ -54,6 +54,8 @@
 
   String ADMINISTER_SECURITY_ACTION = "org.pentaho.security.administerSecurity"; //$NON-NLS-1$
 
+  String MODIFY_DATABASE_ACTION = "org.pentaho.platform.dataaccess.datasource.security.manage"; //$NON-NLS-1$
```

Comment:
This should be `static final`. Please fix others.

Reply:
RepositorySecurirtyProvider is an Interface not a Class, and so, "static final" should not be used (it's the default for Interfaces and Sonar would probably flag it).